CHANGELOG.md

```diff
@@ -63,6 +63,7 @@
 * [ENHANCEMENT] Distributor: improve performance ingesting OTLP payloads. #5531 #5607 #5616
 * [ENHANCEMENT] Ingester: optimize label-values with matchers call when number of matched series is small. #5600
 * [ENHANCEMENT] Compactor: Delete bucket-index, markers and debug files if there are no blocks left in the bucket index. This cleanup must be enabled by using `-compactor.no-blocks-file-cleanup-enabled` option. #5648
+* [ENHANCEMENT] Ingester: reduce memory usage of ActiveSeriesTracker. #5665
```

Comment:
[nit] Just make it human readable, instead of the internal struct name.

```suggestion
* [ENHANCEMENT] Ingester: reduce memory usage of active series tracker. #5665
```